src/index.scss

```diff
@@ -14,7 +15,13 @@ body {
   overflow: hidden;
 }
 
-input, 
-textarea { outline: none; }
+input,
+textarea,
+button { outline: none; }
 
 #root { height: 100%; }
+
+input {
+  background-color: $gridTileDark;
+  border: 1px solid #454a4e;
```

Comment:
is `#454a4e` a defined color variable or a one off?

Reply:
It is not defined in a color variable file.